src/main/webapp/WEB-INF/jsp/login.jsp

```diff
@@ -0,0 +1,28 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <title>Login | Personal Dashboard</title>
+        <link rel="stylesheet" type="text/css" href="${pageContext.request.contextPath}/css/style.css" />
+    </head>
+    <body>
+        <nav id="main-menu">
+        </nav>
+        <section>
+            <div class="container centered">
+              <form action="${pageContext.request.contextPath}/login" method="post">  
+                <label>Userame:</label>
```

Comment:
The `<label>` HTML tag always corresponds to some kind of input field on an HTML form.
In a well-designed form, every input field should have and HTML `id` attribute (aside from the obvious `name` attribute), and every `<label>` should have a `for` attribute with an input field ID as value.

Example:
```html
<label for="user-name">Yo, give me user name</label>
<input id="user-name" name="user-name" type="text"/>
```
Notice that the value of `for` is the same as the input's `id`. This is the link between the two HTML elements.
The purpose is that when a user clicks (or taps, on a mobile device) on the label, the corresponding input field will receive focus.